pkg/controller/installation/typha_autoscaler_test.go

```diff
@@ -117,10 +144,10 @@ func verifyTyphaReplicas(c client.Client, expectedReplicas int) {
 	Eventually(func() int32 {
 		err := test.GetResource(c, typha)
 		Expect(err).To(BeNil())
-		// Just return an invalid number that will never match an expected replica count.
+		// Replicas defaults to 1:  https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas
 		if typha.Spec.Replicas == nil {
-			return -1
+			return 1
 		}
 		return *typha.Spec.Replicas
-	}, 1*time.Second).Should(BeEquivalentTo(expectedReplicas))
+	}, 1).Should(BeEquivalentTo(expectedReplicas))
```

Comment:
Since this is a duration, I think 1 is 1 nanosecond (https://golang.org/pkg/time/#Duration).
Though https://onsi.github.io/gomega/#eventually says that if the number is a float64 then it is interpreted as seconds.
I'm not really sure, are you sure that this is still being interpreted as 1 second. I think I'd prefer to keep it as it was for clarity sake.

Reply:
i believe this was treated properly as 1 second, but agree that we should not have lost that clarity - this has been fixed